dobi.yaml

```diff
@@ -209,3 +209,18 @@ alias=release:
       - 'github-release'
       - 'release-version:capture(DOBI_VERSION)'
       - 'dist-img:push'
+
+
+
+image=dobi:
+    image: test
+    tags: ["latest"]
+    context: .
+#    dockerfile: Dockerfile
+    dobifile:
+      - FROM: busybox
+      - RUN: ls
+      - RUN: mkdir {env.HOME:}
+      - RUN: env
+      - CMD: pwd
```

Comment:
ok, let's not change it yet. I know there have been a couple other projects that have created builders with a yaml config. I'd like to look at them to see what they did. I'll link the projects here when I find them again.